eth_utils/currency.py

```diff
@@ -46,7 +43,7 @@
 MAX_WEI = 2 ** 256 - 1
 
 
-def from_wei(number, unit):
+def from_wei(number, unit, precision=999):
```

Comment:
I don't think this should be configurable.  It should just be a constant, otherwise, bad user decisions lead to imprecise monetary calculations.